app/backend/app.py

```diff
@@ -424,6 +423,19 @@ def create_app():
         # This middleware tracks app route requests:
         app.asgi_app = OpenTelemetryMiddleware(app.asgi_app)  # type: ignore[method-assign]
 
+    if os.getenv("APPLICATIONINSIGHTS_CONNECTION_STRING"):
+        configure_azure_monitor()
+        instrument_app()
+    elif os.getenv("OTLP_GRPC_ENDPOINT"):
```

Comment:
Is this for non-Azure Monitor endpoint?

Reply:
yes, it'll work with all the other vendors